### ISSUE TITLE
guys just copy paste the code and make your own file lol no need to download 

### ISSUE THREAD

**Issue by Kaushal-Teraiya** [apna-college/Delta#50]

**Comment by kavish055** [comment]
hi @Kaushal-Teraiya you are right that copy the code and paste it but my dear brother if anyone face any issue so it's our responsibility to overcome that issue if issue not resolve so that we should inform to our teacher that how he can resolve it.

**Comment by Kaushal-Teraiya** [comment]
> hi @Kaushal-Teraiya you are right that copy the code and paste it but my dear brother if anyone face any issue so it's our responsibility to overcome that issue if issue not resolve so that we should inform to our teacher that how he can resolve it.
>    

Yes you're right but since that teacher is not replying because of other works then we shouldn't waste our time in solving the isse of download when you can simply copy paste 🙂
